06.bowling_object/game.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require './frame'
+
+class Game
+  def initialize(argv = ARGV[0])
+    @marks = argv.split(',')
+  end
+
+  def score
+    frames = build_frames
+    calculate_normal_score(frames) + calculate_bonus_score(frames)
+  end
+
+  private
+
+  def build_frames
+    build_shots.map { |shots| Frame.new(shots) }
```

Comment:
```suggestion
    build_shots.each_slice(2).map { |shots| Frame.new(shots) }
```

こっちでeach_sliceすればto_aは不要になります。

Reply:
ありがとうございます✨修正します！